src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/projects/ProjectSamplesAjaxController.java

```diff
@@ -203,7 +206,13 @@ public ResponseEntity<AjaxResponse> shareSamplesWithProject(@RequestBody ShareSa
 			uiSampleService.shareSamplesWithProject(request, locale);
 			return ResponseEntity.ok(new AjaxSuccessResponse(""));
 		} catch (UIShareSamplesException e) {
-			return ResponseEntity.status(HttpStatus.FORBIDDEN).body(new AjaxErrorResponse(e.getLocalizedMessage()));
+			return ResponseEntity.status(HttpStatus.FORBIDDEN)
+					.body(new AjaxErrorResponse(e.getLocalizedMessage()));
 		}
 	}
+
+	@GetMapping("/files")
```

Comment:
Method is missing javadoc

Reply:
Add in b73814cd22cdf7205611b2c7be1641ee744b7424.  Surprised that was not picked up by our automated testing.